html_app/assignments/sutd_intro_biology_ps2/addmultipledialog.gss

```diff
@@ -0,0 +1,102 @@
+@defmixin gradient(POS, START_COLOR, END_COLOR, FALLBACK_COLOR) {
+  background-color: FALLBACK_COLOR; /* fallback color if gradients are not supported */
+  background-image: -webkit-linear-gradient(POS, START_COLOR, END_COLOR);               /* Chrome 10+,Safari 5.1+ */
+  /* @alternate */ background-image: -moz-linear-gradient(POS, START_COLOR, END_COLOR); /* FF3.6+ */
+  /* @alternate */ background-image: -ms-linear-gradient(POS, START_COLOR, END_COLOR);  /* IE10 */
+  /* @alternate */ background-image: -o-linear-gradient(POS, START_COLOR, END_COLOR);   /* Opera 11.10+ */
+}
+
+.sutd_ps2_dialog {
+    position: absolute;
+    z-index: 999;
+    top:0px;
+    left:0px;
+    width:800px;
+    height:800px;
+    background-color: #80808080;
+    font-family: 'sourcesanspro-regular';
+    font-size:10pt;
+}
+
+.sutd_ps2_inner_dialog_title_close {
+	
+	padding-top: 0px;
+	float:right;
+	-webkit-appearance: none;
+	font-family:'sourcesanspro-semibold';
+	outline:none;
+	height: 26px;
+	width: 26px;
+	cursor:pointer;
+	border: none;
+	border-radius: 12px;
+	font-size: 14pt;
+	color: transparent;
+	background: transparent;
+	background-image: url('../../../images/header/scb_close_button.png');
+	background-repeat: no-repeat;
+
+    
+}
+
+
+.sutd_ps2_inner_dialog_title_close:hover{
+	background-color: #e58986;
+}
+
+
+
+
+.sutd_ps2_inner_dialog {
+    opacity: 5;
+    width: 75%;
+    height: auto;
+    overflow: auto;
+    margin-top: 12%;
+    margin-left: 12%;
+    position: relative;
+    border: 5px solid white;
+	border-radius: 9px;
+	background-color: #f5f5f5;
+}
+
+.sutd_ps2_inner_dialog_title {
+    font-size:12pt;
+    color:white;
+    margin:0px;
+    margin-bottom:5px;
+    padding: 6px;
+    text-align:left;
+    cursor: move;
+    @mixin gradient(top,#27956c,#189689,#27956c);
+    font-family: 'sourcesanspro-bold';
+}
+
+
+.sutd_ps2_inner_dialog_body {
+    padding:20px;
+    padding-top: 0px;
+}
+
+.sutd_ps2_inner_dialog_body > h1 {
+    font-size: 12pt;
+    font-family: sourcesanspro-regular;
+}
+
+.sutd_ps2_inner_dialog_body > h1 > b {
+    font-size: 12pt;
+    font-family: sourcesanspro-semibold;
+}
+
+
+.sutd_ps2_inner_dialog_cancel {
+    width:115px;
+    margin-left: 10px;
+    margin-right:10px;
+    margin-top: 10px;
+
+}
+
+.sutd_ps2_button_float {
+    display:block;
+}
```

Comment:
This is the way all previous assignments were built. It probably does not have be this way, unless in the assignment builder we let the instructor customize the "Add Sample" dialog.